src/go/k8s/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -471,13 +474,15 @@ type ClusterCondition struct {
 }
 
 // ClusterConditionType is a valid value for ClusterCondition.Type
-// +kubebuilder:validation:Enum=ClusterConfigured
+// +kubebuilder:validation:Enum=ClusterConfigured;ClusterQuiescent
 type ClusterConditionType string
 
 // These are valid conditions of the cluster.
 const (
 	// ClusterConfiguredConditionType indicates whether the Redpanda cluster configuration is in sync with the desired one
 	ClusterConfiguredConditionType ClusterConditionType = "ClusterConfigured"
+	// ClusterQuiescentConditionType indicates that the controller has no outstanding work to do, based on the observedGeneration.
```

Comment:
I know this doesn't follow the naming convention but I think we'll be much better served if this is _not_ prefixed with `Cluster`. If `ClusterConfigured` means the "Redpanda Cluster has been configured", then one would assume that `ClusterQuiescent` means the "Redpanda Cluster is quiescent" which is not the case here. This condition pertains to the controller/reconciler itself. Perhaps `ControllerQuiescent` or `ClusterControllerQuiescent`?